src/all/nhentai/build.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     extName = 'NHentai'
     extClass = '.NHFactory'
-    extVersionCode = 42
+    extVersionCode = 44
```

Comment:
IMO, should only bump this by 1, even if there's another active PR for this source.

Usually not an issue, but if for some reason the PRs can't be merged at the same time- well, then you should look into bumping it.

Reply:
thank you for the pointer